packages/perseus/src/widgets/radio/radio.ts

```diff
@@ -1,119 +1,16 @@
-import {radioLogic, randomizeArrayElements} from "@khanacademy/perseus-core";
+import {radioLogic} from "@khanacademy/perseus-core";
 import _ from "underscore";
 
 import Radio from "./radio.ff";
 import {getUserInputFromSerializedState} from "./util";
 
-import type {RenderProps, RadioChoiceWithMetadata} from "./radio-component";
-import type {PerseusStrings} from "../../strings";
 import type {WidgetExports} from "../../types";
 import type {
     PerseusRadioUserInput,
     PerseusRadioWidgetOptions,
     RadioPublicWidgetOptions,
 } from "@khanacademy/perseus-core";
 
-// Transforms the choices for display.
-const _choiceTransform = (
```

Comment:
I broke these functions apart, moved them to `util`, and added a test suite around them.